tracing/pom.xml

```diff
@@ -38,6 +38,11 @@
             <groupId>org.jboss.logging</groupId>
             <artifactId>jboss-logging-processor</artifactId>
         </dependency>
+        <dependency>
+            <groupId>javax.json.bind</groupId>
+            <artifactId>javax.json.bind-api</artifactId>

```

Comment:
Because this jsonb api is already part of `resteasy-jarxrs` dependencies, so I just directly include it. If it's not acceptable, I can modify the `JsonFormattedRESTEasyTracingInfo.java` to make the JSONB also fully reflective loaded to remove this dependency.